### DISCUSSION TITLE
Option to adjust button corner radius

### DISCUSSION THREAD

**Discussion by Schweeeeeeeeeeeeeeee** [ppy/osu#21580] [Ideas] [answered]
I'd like an option to adjust the corner radius on buttons, or an option to revert the button's corner radius to the one it was before the December update.
Example:
![Corner-Radius-Diagram](https://user-images.githubusercontent.com/88291316/206486773-d0ded4e1-f679-4abf-8091-63fcfe367389.png)

Because it looks kind of odd sometimes.
Example:
![image](https://user-images.githubusercontent.com/88291316/206484293-fae7b2de-5566-4491-b86b-d3ca523aac2d.png)
I think they seem way too separated, I don't know how to say it xD

This is just a suggestion and my own opinion, the rest of the December update was just pure awesome and I appreciate how much work you guys put into it.

**Comment by integerrr** [comment, marked as answer]
thats more of an overlook on design/implementation of the buttons, this should be fixed on the design level rather than throwing in an(other) option imo

**Comment by peppy** [comment]
I was aware of this. The countdown button needs special attention to bring it up to the correct design.
